### PR TITLE
Tar vulnerability: run npm audit fix

### LINKED ISSUE #360: High severity vulnerability
`npm-audit` reveals `elm-test` has some vulnerable dependencies. Would be nice to get them upgraded so we can get rid of this error:

```
$ npm init -y && npm install --save elm-test && npm audit

...

+ elm-test@0.19.0-rev6
added 378 packages from 208 contributors and audited 2388 packages in 10.462s
found 1 high severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details
                                                                                
                       === npm audit security report ===                        
                                                                                
# Run  npm install elm-test@0.19.0  to resolve 1 vulnerability
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Arbitrary File Overwrite                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ tar                                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ elm-test                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ elm-test > fsevents > node-pre-gyp > tar                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/803                             │
└───────────────┴──────────────────────────────────────────────────────────────┘


found 1 high severity vulnerability in 2388 scanned packages
  run `npm audit fix` to fix 1 of them.

```

### PR DESCRIPTION
**Pull request by mgold**
Fixes #360 if I've done it right.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -954,12 +973,26 @@
       }
     },
     "elm": {
-      "version": "0.19.0-bugfix6",
-      "resolved": "https://registry.npmjs.org/elm/-/elm-0.19.0-bugfix6.tgz",
-      "integrity": "sha512-nD/oK/nG26ULh94GREsILA9aQa3Gon+KKBmS0Z0MstywLWwrhuWKba8gfiIWLdnpQWHFIf+UsThk6Z71UWi6TQ==",
+      "version": "0.19.0",
```
Comment (avh4) [inline package-lock.json]: You went backwards on the elm version -- This should be "0.19.0-no-deps" for the latest version

Comment (mgold) [reply package-lock.json]: Changed to the suggested string in `package.json` and ran `npm install`.

**Comment by avh4** [comment]
This change looks fine to merge!

But regarding #360, I don't think this is going to fix it -- the path reported in the audit is `elm-test > fsevents > node-pre-gyp > tar`, which suggests that the dependency on `fsevents` is the root cause of the alert -- I think updating it (1.2.4 -> 2.0.6) will probably be necessary to fix #360 (looks like v1.2.8+ fixes the dependency alert: https://github.com/fsevents/fsevents/issues/262)

**Comment by mgold** [comment]
It appears that appveyor, but not travis, is looking at a failing test, expecting it to fail, but thinking it has passed.

**Comment by harrysarson** [comment]
Is the `fsevents` package actually used? A search (<https://github.com/rtfeldman/node-test-runner/search?q=fsevents>) shows only the entry in `package.json`.

**Pull request closed without merging**